test/library/standard/Random/choiceTest.chpl

```diff
@@ -0,0 +1,150 @@
+use Random;
+
+config const debug = false;
+
+proc main() {
+  var pcg = makeRandomStream(real, algorithm=RNG.PCG);
+  runTests(pcg);
+  //var npb = makeRandomStream(real, algorithm=RNG.NPB);
+  //runTests(npb);
```

Comment:
If this is commented out because it doesn't work, let's make the choice function return an error for NPB